Sources/SectionLabel.swift

```diff
@@ -1,16 +1,10 @@
-//
-//  SectionLabel.swift
-//  Demo
-//
-//  Created by Marijn Schilling on 06/06/16.
-//
-//
-
 import UIKit
 
 class SectionLabel: UIView {
 
-    static let RightOffsetForSectionLabel : CGFloat = 90.0
+    static let RightOffsetForActiveSectionLabel: CGFloat = 80.0
+    static let RightOffsetForInactiveSectionLabel: CGFloat = 60.0
+
```

Comment:
Will hear about design specifics this afternoon probably, will check with Kristian if it has to be relative or absolute :v: